packages/nx-python/README.md

```diff
@@ -350,6 +350,33 @@ To identify if the package is publishable, the executor checks `project.json` fi
 
 If the `publish` option is set to `false` and the `--bundleLocalDependencies=false` option is used, the executor will bundle the package.
 
+###### Custom source specification
+
+In addition when adding dependencies in this way its also possible to configure a custom source for a package. This works similar to the `publish` option in that its specified on the target dependencies build options. To use this set the `customSourceName` and `customSourceUrl` to valid values for the source to retrieve the package from for each package stored on a custom Pypi.
+
+`project.json` example:
+
+```json
+{
+  ...
+  "targets": {
+    ...
+    "build": {
+      "executor": "@nxlv/python:build",
+      "outputs": ["apps/myapp/dist"],
+      "options": {
+        "outputPath": "apps/myapp/dist",
+        "publish": false,
+        "customSourceName": "example",
+        "customSourceUrl": "http://example.com/"
+      }
+    },
+  }
+}
+```
+
+Alternatively its also possible to configured it within the `nx.json` as `targetDefaults` across the whole repository.
```

Comment:
This statement is not valid anymore when you read the project config using `getProjectConfig` function, you only can see what is explicitly defined in the `project.json` file.

Reply:
Ah I see, perhaps I should add logic to also check the `nx.json` for the settings then. I think so most use cases it would be preferable to configure a single repo across the whole project.

Reply:
or, maybe we should check if the dependency build options contain the `customSourceName` and `customSourceUrl`, if yes, use it, if no, use from `this.options`, in this way we can use from `nx.json` and CLI.

What do you think?

Reply:
Ok, I have pushed a new commit with this fix and a new test now.

Reply:
Nice, have you tested somewhere else apart from the unit test?

Reply:
Actually I forgot. I tested this locally before and `getProjectConfig` does infact return the default values too. The last change wasn't needed and I'm going to roll it back out.

Reply:
really, so, if the dependency does not have the `customSourceName` and `customSourceUrl` does the `getProjectConfig` actually consider the `nx.json` default values?

Reply:
It does from my testing.

Reply:
that's cool, I didn't know that.

Reply:
could you test using the CLI options, please?

Reply:
It doesn't work via the CLI options because the `customSourceName` and `customSourceUrl` are only used from the target package and not the current package. So it must be set in either the `project.json` or the `nx.json`.

Reply:
Ok then,